src/main/resources/util/thresholds.jelly

```diff
@@ -130,14 +130,14 @@
             </tbody>
           </table>
         </f:entry>
-        <f:entry title="${%Use delta for new warnings}" description="${%description.useDeltaValues}">
+        <f:entry title="${%title.useDeltaValues}" description="${%description.useDeltaValues}">
           <f:checkbox name="useDeltaValues" checked="${instance.useDeltaValues}" />
         </f:entry>
         <f:entry title="${%title.usePreviousBuildAsReference}" description="${%description.usePreviousBuildAsReference}">
           <f:checkbox name="usePreviousBuildAsReference"
             checked="${instance.usePreviousBuildAsReference}" />
         </f:entry>
-        <f:entry title="${%Only use stable builds as reference}"
+        <f:entry title="${%title.onlyUseStableBuildsAsReference}"
```

Comment:
True. I'll simply drop this commit as it's too involving for me to update all translations.